ops/main.py

```diff
@@ -391,8 +400,10 @@ def main(charm_class: Type[ops.charm.CharmBase],
     else:
         actions_metadata = None
 
+    breaking_relation_id = _get_breaking_relation_id()
+
     meta = CharmMeta.from_yaml(metadata, actions_metadata)
-    model = ops.model.Model(meta, model_backend)
+    model = ops.model.Model(meta, model_backend, breaking_relation_id)
```

Comment:
I know it's not required, but let's pass in optional arguments using their keywords, to avoid accidental breakage if we add more and change the order. So `Model(meta, model_backend, breaking_relation_id=breaking_relation_id)`.